src/k8s-engine/wsl.ts

```diff
@@ -307,6 +308,39 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
         this.emit('service-changed', services);
       });
       this.activeVersion = desiredVersion;
+
+      // Temporary workaround: ensure root is mounted as shared -- this will be done later
+      // Right now the builder pod needs to be restarted after the remount
+      try {
+        await childProcess.exec('wsl --user root -d k3s mount --make-shared /');
+        console.log('Waiting for ensuring root is shared');
+        await util.promisify(setTimeout)(60_000);
+        await childProcess.execFile(resources.executable('kim'), ['builder', 'install', '--force', '--no-wait']);
```

Comment:
`execFile` does not return a `Promise`, you can't `await` on it. (Use `util.promisify`)

Reply:
done